README.md

```diff
@@ -131,5 +170,6 @@ role, goal, and reason for each.
 Finally, link to your repo on github in the space below.
 
 Your Answer:
-```text
+```
+https://github.com/beckybeauchamp1/Memory-Game/tree/master
```

Comment:
This is good, but make sure your user stores have a reason as well as a role and a goal!